main/main.c

```diff
@@ -854,16 +855,25 @@ PHPAPI ZEND_COLD void php_log_err_with_severity(char *log_message, int syslog_ty
 			char *tmp;
 			size_t len;
 			zend_string *error_time_str;
+			size_t error_date_format_len;
+			char *error_date_format = PG(error_date_format);
+
+			if (error_date_format == NULL) {
+				error_date_format = "d-M-Y H:i:s e";
+			}
+
+			error_date_format_len = strlen(PG(error_date_format));
```

Comment:
typo `error_date_format_len = strlen(error_date_format);` ?

Reply:
@KalleZ yes you are right, i refactored to use `OnStringUnempty` now as this has the same effect but nicely on the INI level already.